packages/components/src/components/es-empty-state/usage/example.md

```diff
@@ -0,0 +1,16 @@
+```tsx
+
+export default () =>
+<div>
+    <es-empty-state header={'Example empty state'} body={<es-button>{'There are no items'}</es-button>}>
+    <p slot='illustration'>{'Illustration here'}</p>
+    <p slot='foot'>{ 'Footer goes here' }</p>
+    </es-empty-state>
+    <hr/>
+    <es-empty-state header={'Example empty state'} body={'No items to show'} layout='horizontal'>
+    <p slot='illustration'>{ 'Illustration here' }</p>
+    <p slot='foot'>{ 'Footer goes here' }</p>
+    </es-empty-state>
+</div>
+        
```

Comment:
These are usually "recommended usage", so it would be nice to show it in an actual recommended way of using it,  rather than with placeholders.

`illustrations` are available in the docs, so can use this example from cloud:

```suggestion
export default () => (
    <y-empty-state
        header={'Create a new group'}
        body={
            'Creating a new group will allow for the grouping of members with the same access.'
        }
    >
        <es-illustration-group slot={'illustration'} />
        <es-button-link
            slot={'foot'}
            variant={'outline'}
            url={'/new'}
        >
            {'New group'}
            <es-icon icon={'plus'} slot={'after'} />
        </es-button-link>
    </y-empty-state>
);
```;